sdcflows/interfaces/fmap.py

```diff
@@ -531,3 +552,22 @@ def _delta_te(in_values, te1=None, te2=None):
             'EchoTime2 metadata field not found. Please consult the BIDS specification.')
 
     return abs(float(te2) - float(te1))
+
+
+def au2rads(in_file, newpath=None):
+    """Convert the input phase difference map in arbitrary units (a.u.) to rads."""
+    im = nb.load(in_file)
+    data = im.get_fdata().astype('float32')
```

Comment:
Not a big deal for a single volume, but this can avoid casting to float64 before casting to float32.

```suggestion
    data = im.get_fdata(dtype='float32')
```